src/data/docs/documentation.md

```diff
@@ -5,257 +5,232 @@ description: How to write Unison documentation and associate it with your code
 
-### Include the full contents of another Doc value
+# Suggested conventions
 
-To nest a `Doc` value inside another `Doc` value, use `@[include]` followed by
-a name given to that `Doc` value in your codebase. When Unison displays the `Doc`, the entire contents of the included `Doc` will be displayed in that spot.
+Although documentation values don't require any particular structure you might try structuring your docs in accordance with the following guidelines: 
 
-The rule about [transclusion](https://en.wikipedia.org/wiki/Transclusion) applies here too; if the referenced `Doc` value changes, your `Doc` changes automatically as a result.
-
-Note that Unison will not let you create a cycle where you include a `Doc` inside of itself.
-
-### Include any Unison value from the codebase
-
-Sometimes you want to include the result of evaluating some Unison expression. To do this, use `@[evaluate]` followed by the name of a value in your codebase. For example, given a definition like this:
-
-```unison
-exampleExpression = 2 + 2
-```
-
-We'll have to first add it to the codebase:
-
-```ucm
-.> add
-
-  ⍟ I've added these definitions:
-
-    exampleExpression : Nat
-```
-
-Once the definition is in the codebase, we can insert the result of evaluating it
-into any `Doc` value:
-
-```unison
-exampleDoc =
-  [: The value is @[evaluate] exampleExpression :]
-```
-
-When Unison displays `exampleDoc`, this will get expanded to:
+- Start with a brief overview, then maybe a longer description, then some examples, then link to related definitions and further reading.
+- Use Markdown syntax for formatting. Indent examples by 4 spaces for readability when viewing the docs in plain text form, and this will also render nicely as Markdown.
+- Follow sensible naming conventions for associated documentation and examples. For a definition `Jabberwock.whiffle`, for example:
+    - Its primary documentation should be called `Jabberwock.whiffle.doc`, and secondary docs could be in the `Jabberwock.whiffle.docs` namespace. Perhaps a document called `Jabberwock.whiffle.docs.advancedUsages` could show advanced usages of the function.
+    - Examples could be in the `Jabberwock.whiffle.examples` namespace. For instance: `Jabberwock.whiffle.examples.ex1` and `Jabberwock.whiffle.examples.ex2`.
+
+# Evaluating Code 
+We'll reinforce and expand upon the myriad ways Unison source code can be included documentation next. We've observed that Unison source code can be included in docs using the `@source` block. Gone are the days when your docs describe a function which has changed its name or content. Unison docs do not drift from their code definitions!  We've also seen `@eval` for evaluating expressions, and term linking `[namedLink]({myTerm})` but the Unison doc format has additional capabilities. 
```

Comment:
Use "many" instead of "myriad". Remember that a lot of our readers are not native speakers of English.